Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/CancelSearch.hs

```diff
@@ -50,17 +50,17 @@ lockSearchRequest ::
   ) =>
   Id SR.SearchRequest ->
   m ()
-lockSearchRequest searchId = do
-  let searchLockKey = mkSearchLockKey searchId
-      startedCancelInfomKey = mkStartedCancelInfomKey searchId
+lockSearchRequest searchRequestId = do
+  let searchLockKey = mkSearchLockKey searchRequestId
+      startedCancelInfomKey = mkStartedCancelInfomKey searchRequestId
   val <- Hedis.incr searchLockKey
-  cancellingSearchRequest <- fromMaybe False <$> Hedis.get startedCancelInfomKey
+  cancellingSearchReq <- fromMaybe False <$> Hedis.get startedCancelInfomKey
   Hedis.expire searchLockKey 120
-  when (val > 1 && not cancellingSearchRequest) $ throwError (InternalError "FAILED_TO_CANCEL_SEARCH_REQUEST")
+  when (val > 1 && not cancellingSearchReq) $ throwError (InternalError "FAILED_TO_CANCEL_SEARCH_REQUEST")
   Hedis.setExp startedCancelInfomKey True 120
 
 mkStartedCancelInfomKey :: Id SR.SearchRequest -> Text
-mkStartedCancelInfomKey searchId = "SearchRequest:Cancelling:" <> searchId.getId
+mkStartedCancelInfomKey searchRequestId = "SearchTry:Cancelling:" <> searchRequestId.getId
```

Comment:
> Again renaming key can cause issue.

How? You don't have 2 backends running at the same time, right? It will just use new space in redis, when you run upgraded backend

Reply:
This issue will be temporary. For example a ride started before the release and end after the release.

Reply:
> This issue will be temporary. For example a ride started before the release and end after the release.

This is a lock for cancell EP to not run it twice. I don't see how this renaming can break anything, since you'll need to restart server to update it => cancellation request won't be processed